src/components/interfaces/HMI_API.xml

```diff
@@ -3603,6 +3603,23 @@
     </function>
     <function name="DialNumber" messagetype="response">
     </function>
+    <function name="CloseApplication" messagetype="request">
```

Comment:
@IGapchuk shouldn't request contain app_id?
Please add @KhrystynaDubovyk  to review